src/dopri.c

```diff
@@ -110,6 +111,7 @@ dopri_data* dopri_data_copy(const dopri_data* obj) {
   ret->step_size_max = obj->step_size_max;
   ret->step_size_initial = obj->step_size_initial;
   ret->step_max_n = obj->step_max_n;
+  ret->step_size_min_allow = false;
```

Comment:
Should false be obj->step_size_min_allow ?

Reply:
Thanks Wes, great catch